src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/armor/ElytraCap.java

```diff
@@ -0,0 +1,19 @@
+package io.github.thebusybiscuit.slimefun4.implementation.items.armor;
+
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+import org.bukkit.inventory.ItemStack;
+
+/**
+ * A class for the Elytra Cap.
+ *
+ * @author Seggan
+ */
+public class ElytraCap extends SlimefunItem {
```

Comment:
Also you could try to use the SlimefunArmorPiece instead.

Reply:
why?

Reply:
> why?

Because it's entire purpose is sf armor? And using a centralized api is always easier to maintain? Also it has some performance improvements? Also its use should be more wide-spread?

Reply:
okokok

Reply:
> okokok

But this isn't a necessity. Btw you could do this by adding a new ProtectionType called CrashProtection. I'm more than willing to help you in #programming-help channel.

Reply:
thnaks